src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -3369,6 +3370,82 @@ private static void WriteTooLargeHelper(JsonWriterOptions options, ReadOnlySpan<
             jsonUtf8.Flush();
         }
 
+        [ConditionalTheory(nameof(IsX64))]
+        [OuterLoop]
+        [InlineData(true, true)]
+        [InlineData(false, true)]
+        [InlineData(true, false)]
+        [InlineData(false, false)]
+        public void WriteTooLargeArguments(bool formatted, bool skipValidation)
```

Comment:
This test should run relatively quickly, correct, given the check happens right up front?

I am fine it running in outerloop (given it allocates quite a bit), but just checking that runtime wise it wouldn't cause a significant time cost.

Reply:
Yes. Memory allocation is the reason, following example of existing tests with similar setup.